src/mame/drivers/minitel_2_rpic.cpp

```diff
@@ -100,6 +102,7 @@ class minitel_state : public driver_device
 	required_device<i80c32_device> m_maincpu;
 	required_device<ts9347_device> m_ts9347;
 	required_device<palette_device> m_palette;
+	optional_device<i2cmem_device> m_i2cmem;
```

Comment:
Can you make a derived class for systems with the serial EEPROM to avoid the need for an optional device?

Reply:
I just turned it as "required_device". Since this is present in all Minitel 2 there is no reason to keep it optional.
Fixed.